includes/data-stores/class-wc-admin-reports-orders-data-store.php

```diff
@@ -279,8 +285,12 @@ protected function include_extended_info( &$orders_data, $query_args ) {
 				'products'   => array(),
 				'categories' => array(),
 				'coupons'    => array(),
+				'customer'   => array(),
```

Comment:
I was a bit surprised that there wasn't any test coverage in the [API test suite](https://github.com/woocommerce/wc-admin/blob/master/tests/api/reports-orders.php) around `extended_info` outside of verifying the schema. Might be nice to queue up an issue to add in some more test coverage for the extended_info

Reply:
Issue created [here](https://github.com/woocommerce/wc-admin/issues/1596).